ts/features/messages/components/MessageAttachmentPreview.tsx

```diff
@@ -129,9 +131,10 @@ type Props = {
 export const MessageAttachmentPreview = (props: Props): React.ReactElement => {
   const [isError, setIsError] = useState(false);
 
-  const attachmentId = props.attachmentId;
+  const messageId = props.messageId;
+  const id = props.attachmentId;
```

Comment:
```suggestion
  const atrachmentId = props.attachmentId;
```
Maybe it's better to leave it with the full name what do you think?

Reply:
Fixed b8be121df21c02d8931503533f7ce5119c363d12